requirements/production.txt

```diff
@@ -1,6 +1,5 @@
 -r base.txt
 django-celery==3.1.1
 gunicorn==18.0
-redis==2.9.1
-django-redis-cache==0.10.2
```

Comment:
I've swapped out ``django-redis-cache`` for ``django-redis`` as it's better maintained and has more features (we've done this changes on most Torchbox sites as well).

Wagtaildemo doesn't use ``redis`` directly so removing it from here lets ``django-redis`` pull in the version it thinks is best with its dependencies.